package.json

```diff
@@ -24,5 +24,8 @@
   "scripts": {
     "fixlint": "npm run fixlint --prefix client && npm run fixlint --prefix server && prettier --write '**/*.js'",
     "lint": "npm run lint --prefix client && npm run lint --prefix server && prettier --check '**/*.js'"
+  },
+  "dependencies": {
+    "cipm": "^2.0.0"
```

Comment:
Curious about cipm. Is this necessary?

Reply:
Nice catch. I'll check if it is at all needed. Could be one of my tweaking attempts while I was trying to make build work. 

Vanilla repo wouldn't build on my local machine, because apparently some of the dependencies in the ```package-lock.json``` could not be properly installed. I think one of the suggestions I found online referred to ```cipm``` to be used, so I added it as a dependency. But I don't think it helped.

After all, what actually fixed the build for me was removing package-lock.json altogether and re-generating it with ```npm install```.

Reply:
You can run `npm ci` in the directory of a package-lock.json to install those dependencies. There are 3 package-lock files in this project... `scripts/build.sh` installs all of them. I should probably add those separate steps in the README

Reply:
Well, ```scripts/build.sh``` wouldn't install them for some reason. Also, I seem to have to run it twice for it to do all the right things, so there's definitely something wrong with that. Or maybe it's just Windows...

Reply:
But yes, ```cipm``` is not needed here. Will remove.